Assets/Plugins/Source/Core/Config.cs

```diff
@@ -280,22 +280,16 @@ protected virtual async Task ReadAsync()
         /// </summary>
         protected virtual void Read()
         {
-            bool configFileExists = File.Exists(FilePath);
-
-            if (configFileExists)
+            
+#if UNITY_EDITOR // This should only happen in Editor Runtime (Play Mode)
+            if (!File.Exists(FilePath)) 
```

Comment:
During runtime when not in the editor - how do Config files get read?

I think most of them utilize `ReadAsync`, but I think that's just a coincidence.

Reply:
During runtime on any platform (including editor), it does use this `Read()`
The comment here is talking about the `Write`.
I'll add a couple words to make it clear.

ReadAsync is used in editor when not in runtime (i.e. in EditorsWindows)